cmd/stack/init.go

```diff
@@ -24,7 +24,12 @@ func Init(cliCtx *cli.Context) error {
 		}
 	}
 
-	projectTitle, err := prompts.ProjectTitle("", false)
+	projectTitle := ""
+	if cliCtx.IsSet("project") {
+		projectTitle = cliCtx.String("project")
+	}
+
+	projectTitle, err = prompts.ProjectTitle(projectTitle, false)
```

Comment:
in the other commands it will skip a prompt if a field is set as a way to automate